Apps/WebClient/src/ClientApp/src/components/modal/AddQuickLinkComponent.vue

```diff
@@ -165,8 +190,8 @@ export default class AddQuickLinkComponent extends Vue {
     >
         <form data-testid="quick-link-modal-text">
             <b-row
-                v-for="(quickLink, index) in enabledQuickLinkFilter"
-                :key="index"
```

Comment:
Why was the :key value changed?

Reply:
When Mike and I were going over the strategy for my changes he noticed that and suggested the change to use quicklink.module instead so that it is a unique value to that item only. You may want to check it with him.